package.json

```diff
@@ -49,17 +49,14 @@
 		"storybook": "start-storybook  -c ./storybook -p 6006 --ci",
 		"storybook:build": "build-storybook  -c ./storybook -o ./storybook/dist",
 		"test": "wp-scripts test-unit-js --config tests/js/jest.config.json",
-		"test:e2e:down": "docker-compose down",
-		"test:e2e:down:reset": "docker-compose down -v",
-		"test:e2e:ssh": "docker exec -it woo-blocks_wordpress-www_1 /bin/bash",
-		"test:e2e:up": "docker-compose up -d",
-		"test:e2e:up:build": "docker-compose up --build -d --quiet-pull",
```

Comment:
We should probably document later how can you start and stop the docker instance
`npm run wp-env start` `npm run wp-env stop`

Reply:
Agreed, and likely we can just point to the package docs (so folks can see all the other stuff they can do beyond `start` and `stop`)